qiskit_nature/transformers/particle_hole_transformer.py

```diff
@@ -28,5 +45,2102 @@ def transform(self, q_molecule: QMolecule) -> QMolecule:
         Returns:
             A new `QMolecule` instance.
         """
-        # TODO
-        raise NotImplementedError()
+
+        n_qubits = self.num_orbitals * 2
+
+        self._h1 = q_molecule.one_body_integrals
+        self._h2 = q_molecule.two_body_integrals
+
+        print(self._h1)
+
+        self._convert_to_interleaved_spins()
+
+        print(self._h1)
+
+        h1_old_matrix = self._h1
+        h2_old_matrix = self._h2
+
+        if q_molecule.mo_onee_ints_b is not None:
+            raise QiskitNatureError("Particle Hole Transformer does not work with UHF")
```

Comment:
The ParticleHole from before was changed to work with open shell right. This test https://github.com/Qiskit/qiskit-aqua/blob/master/test/chemistry/test_particle_hole.py included `ROHF` and well as `UHF` tests. I think that above test would exclude all open shell usage,.

Reply:
This was removed by @mrossinek. This is fixed.